lib/bamboo.js

```diff
@@ -0,0 +1,17 @@
+// https://confluence.atlassian.com/bamboo/bamboo-variables-289277087.html
+
+module.exports = {
+  detect() {
+    return Boolean(process.env.bamboo_agentId);
+  },
+  configuration() {
+    return {
+      service: 'bamboo',
+      commit: process.env.bamboo_planRepository_1_revision,
```

Comment:
I know this isn't camelCase, but Bamboo doesn't give these env vars as uppercase.... Can we ignore this with `xo`?

Reply:
`xo` works with ESLint. You can use the same type of comment to disable a rule: https://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments